vscode/snippets/snippets-gen/src/main/java/org/wso2/integration/ballerina/SnippetsContentGenerator.java

```diff
@@ -64,20 +65,31 @@ static void generateSnippetContent(List<Snippet> snippetList) throws IOException
 
         String snippetContent = "";
 
-        for (Snippet snippetObject : snippetList) {
+        for (int i = 0; i < snippetList.size(); i++) {
             StringBuilder stringBuilder = new StringBuilder();
-            Pair<String, String> pair = getImportText(snippetObject);
+            Pair<String, List<String>> pair = getImportText(snippetList.get(i));
```

Comment:
Shall we use a Map rather than a Pair? Pair is coming from javafx...

Reply:
This is addressed in  in https://github.com/wso2/ballerina-integrator/issues/441